frodo.js

```diff
@@ -2,130 +2,124 @@
  * Extras Plugin
```

Comment:
SCRIPT-This new feature lacks unit tests. Could you add some to ensure everything works correctly?